image_utils.py

```diff
@@ -689,7 +689,7 @@ def np_to_render_pass(
         array = to_dtype(array, dtype)
     if top_to_bottom:
         array = np.flipud(array)
-    render_pass.rect.foreach_set(array.reshape(-1, render_pass.channels))
+    render_pass.rect.foreach_set(array.reshape(-1))
```

Comment:
```suggestion
    if BLENDER_VERSION >= (4, 1, 0):
        render_pass.rect.foreach_set(array.reshape(-1))
    else:
        render_pass.rect.foreach_set(array.reshape(-1, render_pass.channels))
```
Looks like 3.6 and 4.0 (untested for anything earlier but I expect the same) require 2D, and 4.1 requires 1D.

Reply:
Ah, that explains it. Thanks for checking on that.